src/jit/emitxarch.cpp

```diff
@@ -5249,6 +5249,38 @@ void emitter::emitIns_AR_R(instruction ins, emitAttr attr, regNumber ireg, regNu
     emitAdjustStackDepthPushPop(ins);
 }
 
+//------------------------------------------------------------------------
+// emitIns_S_R_I: emits the code for an instruction that takes a stack operand,
+//                a register operand, and an immediate.
+//
+// Arguments:
+//    ins       - The instruction being emitted
+//    attr      - The emit attribute
+//    varNum    - The varNum of the stack operand
+//    offs      - The offset for the stack operand
+//    reg       - The register operand
+//    ival      - The immediate value
+//
+void emitter::emitIns_S_R_I(instruction ins, emitAttr attr, int varNum, int offs, regNumber reg, int ival)
+{
+    assert(ins == INS_vextracti128 || ins == INS_vextractf128);
+    instrDesc* id = emitNewInstrAmdCns(attr, 0, ival);
```

Comment:
`ival` should just be 0 or 1 here (for now anyways), maybe assert?

Reply:
Will do.